### ISSUE TITLE
DESIGN - Update slider headings

### ISSUE THREAD

**Issue by fionamorrison23** [civictheme/uikit#888]
### User story

As an anonymous user
I want the heading hierarchy updated on the slider
So that it conforms to WCAG 2.2 accessibility guidelines

### Description

The Slider headings are marked up incorrectly as per https://github.com/civictheme/uikit/issues/878.
Update the headings in the design   so they're semantically correct.
The code solution is going to be:
When there is a main component title, it will be H3. And in that case, the slider card titles are H3.
But then there is no main component heading, the slider card titles are H3.
This should be reflected in Figma in some way. 

### Acceptance criteria

**AC 1 - Main component heading**
Given I am an anonymous user
When I view a Slider component
And that component has an overall component heading
Then the heading is H2
And the slider card title is H3

**AC 2 - No main component heading**
Given I am an anonymous user
When I view a Slider component
And that component does not have an overall component heading
Then the slider card title is H3
